### PR TITLE
fix constraint pivot

### PR DESCRIPTION
**Pull request by wpride**
http://manage.dimagi.com/default.asp?176697#982290

If there is more than one pivot (IE both a > and <) we should just not use the hint because they're likely opposing

### CODE REVIEW

**Comment by ctsims** [comment]
Will:

Can you add a bunch of tests around the pivots that illustrate the thing that was breaking?

**Comment by wpride** [comment]
@ctsims

File: core/src/test/resources/test_constraints.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                    <constraint-min />
+                    <constraint-max-and-min />
+                    <constraint-max-or-min />
+                </data>
+            </instance>
+
+            <bind nodeset="/data/constraint-max" constraint=". < 30" type="xsd:int"/>
+            <bind nodeset="/data/constraint-min" constraint=". > 10" type="xsd:int"/>
+            <bind nodeset="/data/constraint-max-and-min" constraint=". < 100 and . > 300" type="xsd:int"/>
+            <bind nodeset="/data/constraint-max-or-min" constraint=". < 30 or . < 100" type="xsd:int"/>
```
Comment (ctsims) [inline core/src/test/resources/test_constraints.xml]: This is actually "min or min"

**Comment by ctsims** [comment]
And to confirm, this does (unfortunately) mean that we lose ".> 0 .< 100"
right? That definitely used to work, and must have broken at some point so
it's better to disable it if so, just unfortunate to lose it, since that's
actually about 90% of what hte pivots do (if there's no minimum bound we
can't constrain the input size)

**Comment by wpride** [comment]
fixed and confirmed

**Pull request merged**